benchmarks/build.rs

```diff
@@ -89,6 +89,15 @@ fn main() -> anyhow::Result<()> {
     Ok(())
 }
 
+fn retry<Ok, Err>(fun: impl Fn() -> Result<Ok, Err>, times: usize) -> Result<Ok, Err> {
+    for _ in 0..times {
+        if let ok @ Ok(_) = fun() {
+            return ok;
+        }
+    }
```

Comment:
Don't we want to sleep, here?

Reply:
As you want, I don't kwow why it’s failing

Reply:
I neither know 😂